### PR TITLE
Support celery multi

### PR DESCRIPTION
**Pull request by sakisv**
### Add a start script for running celery multi

The existing script would not work with `celery multi` as it was trying
to put it in the background and the get its pid.~

`celery multi` creates a number of worker processes and stores their
PIDs in files named celeryN.pid, where N the index number of the worker
(starting at 1).

### Make the sender-worker use 4 celery workers and 2G of RAM

### CODE REVIEW

**Comment by sakisv** [comment]
We need to test this before it gets merged.

File: scripts/run_multi_worker_app_paas.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  fi
+
+  if [ -z "${CW_APP_NAME}" ]; then
+    CW_APP_NAME=${NOTIFY_APP_NAME}
+  fi
+}
+
+function configure_aws_logs {
+  # create files so that aws logs agent doesn't complain
+  touch /home/vcap/logs/gunicorn_error.log
```
Comment (richardc0) [inline scripts/run_multi_worker_app_paas.sh]: /home/vcap/logs/ is used a few times - could make it a const so its easier to change

File: scripts/run_multi_worker_app_paas.sh [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+file = /home/vcap/logs/gunicorn_error.log
+log_group_name = paas-${CW_APP_NAME}-gunicorn
+log_stream_name = {hostname}
+EOF
+}
+
+# For every PID, check if it's still running
+# if it is, send the sigterm
+function on_exit {
+  n=0
```
Comment (richardc0) [inline scripts/run_multi_worker_app_paas.sh]: Could use a better descriptor? wait_time? total_wait_time?

File: scripts/run_multi_worker_app_paas.sh [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      if [[ "$n" -ge "$TERMINATE_TIMEOUT" ]]; then
+        echo "Timeout reached, killing process with pid ${APP_PID}"
+        kill -9 ${APP_PID} || true
+        continue
+      else
+        echo "Timeout not reached yet, checking " ${APP_PID}
+        # else, if process is still running send SIGTERM
+        if [[ $(kill -0 ${APP_PID} 2&>/dev/null) ]]; then
+          echo "Terminating celery process with pid ${APP_PID}"
+          kill ${APP_PID} || true
```
Comment (richardc0) [inline scripts/run_multi_worker_app_paas.sh]: Is it better be specific sigterm -15?

File: scripts/run_multi_worker_app_paas.sh (hunk 2 above)
Comment (sakisv) [reply scripts/run_multi_worker_app_paas.sh]: I was just copying from the `run_app_paas.sh`, but ok, I'll change it in both.

File: scripts/run_multi_worker_app_paas.sh (hunk 3 above)
Comment (sakisv) [reply scripts/run_multi_worker_app_paas.sh]: I don't think so, I'd only specify a signal if it's not the default.

**Pull request merged**